paasta_tools/long_running_service_tools.py

```diff
@@ -345,6 +353,19 @@ def get_autoscaling_params(self) -> AutoscalingParamsDict:
             defaults=default_params,
         )
 
+    def validate(self, params: Optional[List[str]] = None,) -> List[str]:
+        error_messages = super().validate()
```

Comment:
should you pass along `params` here?

Reply:
Oh yeah, added that late after mypy complained about it.  I will fix.